### PR TITLE
Adding help creating a collapsable list

### PR DESCRIPTION
**Pull request by Tschuppi81**
Forms: Adds user snippet for subfields to formcode

Feature
LINK: ogc-977

### CODE REVIEW

**Comment by Tschuppi81** [comment]
@Daverball, do you think this is useful for the clients?

**Comment by Daverball** [comment]
@Tschuppi81 Difficult to predict. But I think "Collapsable list" as a name for the concept is misleading, it's more like "Subfields dependent on selected option". It might also be a little bit more helpful to have one example each for checkboxes and radio buttons, otherwise people may get the idea, that only of the two work.

Also just as a side note: It's called formcode and not YAML forms. Formcode is closer to Markdown than YAML. We just happen to transform the Formcode to YAML, so we can use the YAML parser to do some of the heavy lifting for us, rather than having to specify the Formcode grammar completely in pyparsing and manually having to process the ParseResult into the proper structure of fieldsets, fields and subfields.

File: src/onegov/form/parser/snippets.py [hunk 1]
```diff
@@ -39,6 +39,13 @@ class Snippets:
             '    [ ] B\n'
             '    [ ] C'
         )),
+        (_("Subfields depending on selected option"), (
```
Comment (Daverball) [inline src/onegov/form/parser/snippets.py:42]:
```suggestion
        (_("Subfields depending on choice"), (
```

This wording is a little bit more consistent

**Review by Daverball — COMMENTED** [review]
The wording is currently a little bit inconsistent, so I'd slightly adjust it to avoid potential confusion. Otherwise this looks good now.

File: src/onegov/form/parser/snippets.py (hunk 1 above)
Comment (Tschuppi81) [reply src/onegov/form/parser/snippets.py:42]: I like it, will adapt

File: src/onegov/form/locale/de_ch/LC_MESSAGES/onegov.form.po [hunk 2]
```diff
@@ -80,6 +80,9 @@ msgstr "Auswahl"
 msgid "Multiple Choice"
 msgstr "Mehrfachauswahl"
 
+msgid "Subfields depending on choice"
+msgstr "Unterfelder je nach Wahl"
```
Comment (Daverball) [inline src/onegov/form/locale/de_ch/LC_MESSAGES/onegov.form.po:84]:
```suggestion
msgstr "Unterfelder je nach Auswahl"
```

**Review by Daverball — APPROVED** [review]
Minor nit

**Pull request merged**